src/app/game/settings/page.tsx

```diff
@@ -0,0 +1,210 @@
+import NavigationFooter from "@/features/game/NavigationFooter";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function GamePage() {
+  const pageTitle = "設定";
+
+  return (
+    <div className="relative h-full">
+      <h1 className="fixed inset-x-0 top-0 bg-[#0094f4] p-4 pt-7 text-center text-3xl text-white">
+        {pageTitle}
+      </h1>
+
+    <div className="mx-auto justify-center pb-24 pt-28">
+        <div className="mx-auto mb-5 flex w-[90%] items-center">
+            <Image
+                src={"/images/diamond.png"}
+                alt="ダイヤモンド"
+                width={50}
+                height={50}
+                className="mr-5"
+            />
+            <span className="text-3xl">ユーザー設定</span>
+        </div>
+
+        <div className="mx-auto w-3/4">
+            <div className="mb-5 flex items-center">
+                <Image
+                    src={"/images/mail.png"}
+                    alt="メール"
+                    width={30}
+                    height={30}
+                    className="mr-5"
+                />
+
+                <Link href={`/game/settings/account/change/email`}>
+                    <h2 className="text-2xl">メールアドレス変更</h2>
```

Comment:
ネストされた要素についても`span`が良いと思いました。
```suggestion
                    <span className="text-2xl">メールアドレス変更</span>
```
他の部分も同様です。